core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -496,12 +498,16 @@ var ExplorationEditorMainTab = function() {
     // Wait for browser to time out the popover, which is 4000 ms.
     await waitFor.invisibilityOf(
       postTutorialPopover, 'Post-tutorial popover does not disappear.');
-
     await waitFor.elementToBeClickable(
       stateEditContent,
       'stateEditContent taking too long to appear to set content');
-    await stateEditContent.click();
```

Comment:
Now I think the `stateEditContent` variable can be removed from the file since this is its only usage, right?

Reply:
Yep, removed it!